routes/web.php

```diff
@@ -45,6 +45,7 @@
     $router->post('/get-camp-history','CampController@getCampHistory');
     $router->post('/parse-camp-statement', 'StatementController@parseStatement');
     $router->post('/support-and-score-count', 'SupportController@getCampSupportAndCount');
+    $router->get('/get-policy-policy-content','PrivacyPolicyController@getPrivacyPolicyContent');
```

Comment:
Route name can be changed to `get-policy-policy-content`

Reply:
@akuks Done